src/commonMain/kotlin/fr/acinq/eclair/transactions/Transactions.kt

```diff
@@ -94,7 +120,10 @@ object Transactions {
         data class HtlcPenaltyTx(override val input: InputInfo, @Serializable(with = TransactionKSerializer::class) override val tx: Transaction) : TransactionWithInputInfo()
 
         @Serializable
-        data class ClosingTx(override val input: InputInfo, @Serializable(with = TransactionKSerializer::class) override val tx: Transaction) : TransactionWithInputInfo()
+        data class ClaimHtlcDelayedOutputPenaltyTx(override val input: InputInfo, @Serializable(with = TransactionKSerializer::class) override val tx: Transaction) : TransactionWithInputInfo()
+
+        @Serializable
+        data class ClosingTx(override val input: InputInfo, @Serializable(with = TransactionKSerializer::class) override val tx: Transaction, val toLocalOutput: OutputInfo?) : TransactionWithInputInfo()
```

Comment:
Because storage is cheap and it's more convenient, that means when printing channel data you directly see it (instead of having to parse the transaction to look at the right output). We could indeed only keep the index, but we would diverge from `eclair`.

Reply:
Done in https://github.com/ACINQ/eclair-kmp/pull/229/commits/d2860931f8761fa5bb44f0380a3ef63b22b55398